src/main/scala/nl/knaw/dans/easy/fedora2vault/Command.scala

```diff
@@ -40,25 +40,24 @@ object Command extends App with DebugEnhancedLogging {
     .doIfFailure { case NonFatal(e) => println(s"FAILED: ${ e.getMessage }") }
 
   private def runSubcommand(app: EasyFedora2vaultApp): Try[FeedBackMessage] = {
+    lazy val ids = commandLine
+      .datasetId.map(Iterator(_))
+      .getOrElse(commandLine.inputFile()
+        .lineIterator
+        .filterNot(_.startsWith("#"))
+      )
+    lazy val strict = commandLine.strictMode()
+    lazy val writer = commandLine.logFile().newFileWriter(append = true)
+    lazy val outputDir = commandLine.outputDir()
+
     commandLine.transformation() match {
-      case SIMPLE => simpleTransform(app)(SimpleChecker(app.bagIndex))
-      case THEMA => simpleTransform(app)(ThemaChecker(app.bagIndex))
-      case _ => Failure(new Exception(s"transformation type not implemented"))
+      case SIMPLE_SIP =>
+        // TODO construct a SIP in a staging directory, on success move to outputDir
```

Comment:
Configure the staging directory in `application.properties` or on the command line?

Reply:
yes, staging-dir can be configured in the application properties